arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -3751,6 +3951,38 @@
       },
       "description": "The content hash."
     },
+    "RetryHistory": {
+      "type": "object",
+      "properties": {
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the start time."
```

Comment:
I will keep this for now since it is consistent with our other existing API's. Will discuss with PM and we can change all of them as a separate PR. Don't think this is blocking.